packages/docs/docs/docs/first.md

```diff
@@ -0,0 +1,161 @@
+---
+id: first
+title: Your first project
+---
+
+This tutorial will get you started using ZeppelinOS. We will create a new project with a simple contract, deploy it to a development network, interact with it from the terminal, and then update it.
```

Comment:
Given that we have renamed the `update` command to `upgrade`, should we change all these `update` references?